mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -621,6 +621,15 @@ def ROCDL_DPPUpdateOp : ROCDL_IntrOp<"update.dpp", [], [0],
   Arguments<(ins LLVM_Type:$old, LLVM_Type:$src, I32Attr:$dppCtrl, I32Attr:$rowMask,
       I32Attr:$bankMask, I1Attr:$boundCtrl)> {
   let results = (outs LLVM_Type:$res);
+  let summary = "Represent the DPP(Data-Parallel Primitives) functionality";
```

Comment:
This wording is pretty generic and confusing due to that. DPP in general is a broad concept. What "functionality" you mean here? Maybe something like "Exchange data among threads in the same subgroup"?

Nit: `DPP (` (leave a space here)